FluentScreenRecorder/App.xaml

```diff
@@ -3,17 +3,17 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
-    xmlns:local="using:FluentScreenRecorder">
+    xmlns:local="using:FluentScreenRecorder"
+    xmlns:settings="using:Microsoft.PowerToys.Settings.UI">
 
     <Application.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
-                <muxc:XamlControlsResources/>
+                <muxc:XamlControlsResources />
+                <settings:SettingsThemeResources />
                 <ResourceDictionary Source="ms-appx:///Themes/generic.xaml"/>
-            </ResourceDictionary.MergedDictionaries>        
-
+            </ResourceDictionary.MergedDictionaries>
         </ResourceDictionary>
-        
     </Application.Resources>
     
 </Application>
```

Comment:
General comment, but to have good scaling all the ui  widths/heights/paddings have to be multiples of 4, I forgot as well sometimes, but the pr is full of non-multiples and that's bad since in highres displays logical pixels may not correspond to physical ones.